lib/transactions/signature.js

```diff
@@ -1,6 +1,18 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating signature in Lisk Blockchain
```

Comment:
Signatures module provides functions for creating signatures for use with the Lisk Blockchain.